src/middleware/translations/en.json

```diff
@@ -0,0 +1,177 @@
+{
+    "greeting": "Hello",
+    "share-page": "Share this page",
+    "top-of-page": "Back to top",
+    "app-running": "App is running",
+    "health-notes": "Expand endpoint to check for database connection and other services.",
+    "developer_warning": "<strong class=\"govuk-tag govuk-tag--red govuk-phase-banner__content__tag\">Warning</strong> This is a developer page and has not been through user research",
+    "developer_tag": "<span class=\"govuk-tag govuk-tag--yellow govuk-phase-banner__content__tag\">Developer Page</span>",
```

Comment:
When we export this to send to the translators, will we strip the HTML or include it?